bigquery_etl/query_scheduling/task.py

```diff
@@ -269,7 +269,8 @@ def __attrs_post_init__(self):
             self.version = query_file_re.group(4)
 
             if self.task_name is None:
-                self.task_name = f"{self.dataset}__{self.table}__{self.version}"
+                # limiting task name to allow longer dataset names
+                self.task_name = f"{self.dataset}__{self.table}__{self.version}"[:62]
```

Comment:
for sufficiently long dataset names, this would cause all tasks to have the same task name. I think we should instead take the last 62 characters:

```suggestion
                self.task_name = f"{self.dataset}__{self.table}__{self.version}"[-62:]
```

Reply:
oh, I didn't think about that. Yes, you're 100% right. Thanks!